yt/utilities/command_line.py

```diff
@@ -71,6 +72,15 @@ def _fix_ds(arg, *args, **kwargs):
 def _add_arg(sc, arg):
     if isinstance(arg, str):
         arg = _common_options[arg].copy()
```

Comment:
`_common_options` looks potentially undefined. I don't get where it's supposed to come from

Reply:
It is defined weeeeeell below. I am ok to move it, but maybe as part of another PR?

Reply:
sure